MessageManager.js

```diff
@@ -149,6 +149,26 @@ module.exports.define("getString", function (separator, type_filter) {
 
 
 /**
+* purpose: Returns the input prefix concatenated with the this.prefix string
```

Comment:
I know you're just trying to fix something that is broken, but this function was excluded on purpose.
I think that the `addJSON()` function should use `getNewPrefix()` like `addMessagesToStringArray()`